#goal
uint8 ARM_ID_LEFT = 0
uint8 ARM_ID_RIGHT = 1

uint8 arm_id
duration gesture_duration

---
#state

---
#feedback

uint8 MOVING_TO_TARGET = 0
uint8 AT_TARGET = 1
uint8 feedbackState
